scripts/qa_crowbarsetup.sh

```diff
@@ -2656,6 +2656,8 @@ function custom_configuration()
         provisioner)
             # set default password to 'linux'
             proposal_set_value provisioner default "['attributes']['provisioner']['root_password_hash']" "\"$(openssl passwd -1 linux)\""
+            # set discovery root password too
+            iscloudver 6plus && proposal_set_value provisioner default "['attributes']['provisioner']['discovery']['append']" "\" DISCOVERY_ROOT_PASSWORD=linux\""
```

Comment:
Technically, it's only in GMC2+, but I guess I'm okay with that. Also you don't need the space at the beginning of the attribute.